frontend/rust-lib/flowy-database/src/services/field/type_options/date_type_option/date_type_option.rs

```diff
@@ -23,9 +23,6 @@ pub struct DateTypeOptionPB {
 
   #[pb(index = 2)]
   pub time_format: TimeFormat,
-
-  #[pb(index = 3)]
-  pub include_time: bool,
```

Comment:
This property can't be removed even though the property we don't use it anymore. Because the protobuf will use these properties declaration to parser the data. 

If the protobuf struct doesn't need to parser from the disk data, it's ok to remove its properties. For example, the protobuf passthrough the Rust and Dart via the FFI.